sync.cmd

```diff
@@ -0,0 +1,3 @@
+@if "%_echo%" neq "on" echo off
+call executor.cmd %* sync -p
```

Comment:
Should this be `call executor.cmd sync -p %*`? Shouldn't the extra arguments be passed to the sync command?